cmd/acra-server/acra-server.go

```diff
@@ -127,7 +127,8 @@ func realMain() error {
 	apiPort := flag.Int("incoming_connection_api_port", cmd.DefaultAcraServerAPIPort, "Port for AcraServer for HTTP API")
 
 	keysDir := flag.String("keys_dir", keystore.DefaultKeyDirShort, "Folder from which will be loaded keys")
-	keysCacheSize := flag.Int("keystore_cache_size", keystore.InfiniteCacheSize, "Maximum number of keys stored in in-memory LRU cache in encrypted form. 0 - no limits, -1 - turn off cache")
+	cacheKeystoreOnStart := flag.Bool("cache_keystore_on_start", true, "Load all keys to cache on start")
```

Comment:
oh, I missed naming issue. Let's name this new flag in same manner as other keystore related parameters: `keystore_cache_load_on_start`? We try related groups of flags name with similar prefixes. So here it is `keystore_cache_*`. But better I will raise this question in the slack to find consensus name from other colleagues)

Reply:
and we should propagate all these changes to acra-translator too

Reply:
Thanks a lot, already fixed!